css/Style2.css

```diff
@@ -0,0 +1,136 @@
+/* MAIN */
+
+.main-container {
+    display:flex;
+}
+
+.form-section {
+    width: 50%;
+}
+
+.text-param {
+    color:#DE6E49 !important;
+    font-weight: 400;
+    padding-left: 50px;
+    padding-right: 30px;
+    margin-top: 100px;
+    margin-bottom: 60px;
+    font-size: 50px;
+}
+
+
+.main-img {
+    width: 50%;
+}
+
+.img-dimensions {
+    width: 701px;
+    height: auto;
+}
+
+/* FORM */
+
+fieldset { 
+    border: none;
+    width: 30%;
+}
+
+.form-dimensions {
+    display: flex;
+    flex-direction: column;
+    width: 520px;
+}
+
+.form-elem {
+    display: flex;
+    flex-direction: column;
+    margin: 10px;
+}   /* aligns text and input */
+
+.full-name {
```

Comment:
So glad to see the class reusing in the code.
The next level tip here: If a class is reused in multi-locations, we need to come up with a more generic name that describes them all. E.g. `full-name` -> `large-input` or `full-width-input`, ...